variables/collection.go

```diff
@@ -15,6 +15,14 @@ import (
 	specs "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+var ShellOutEnvs map[string]struct{}
+
+func init() {
+	ShellOutEnvs = map[string]struct{}{
+		"HOME": struct{}{},
+	}
+}
```

Comment:
I believe that you can initialize the map without the use of `init`. See e.g. https://stackoverflow.com/a/41078296/1353666